tests/pnpq/devices/test_waveplate_thorlabs_k10cr1.py

```diff
@@ -33,14 +34,26 @@
 from pnpq.errors import InvalidStateException
 from pnpq.units import pnpq_ureg
 
-ustatus_message = AptMessage_MGMSG_MOT_GET_STATUSUPDATE(
+ustatus_message = AptMessage_MGMSG_MOT_GET_STATUSUPDATE_20_BYTES(
```

Comment:
This is not your fault, but we should probably rename this variable, since this is not a ustatus message.